File: Data Structures & Algorithms/concatenation-of-array/submission-1.java
class Solution {
    public int[] getConcatenation(int[] nums) {

        List <Integer> list = new ArrayList<>();
 
        for (int i = 0; i <= 1; i++){

            for (int n : nums){
                list.add(n);
            }


        }

        int [] res = new int[2 * nums.length];

        System.out.println(list);

        for (int i = 0; i < list.size() ; i ++){
            res[i] = list.get(i);
        }
        
        return res;
    }
}

File: Data Structures & Algorithms/is-anagram/submission-1.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        map = {}

        for c in s:
            map[c] = map.get(c , 0) + 1
        
        for c in t:
            map[c] = map.get(c , 0) - 1

        
        for c in map:
            if not map.get(c , 0) == 0:
                return False
        
        return True
        